dipy/boots/resampling.py

```diff
@@ -1,15 +1,21 @@
 #!/usr/bin/python
 
-#import modules
+# import modules
 import time
-import sys, os, traceback, optparse
+import sys
+import os
+import traceback
+import optparse
```

Comment:
@arokem atually if you inspect the modules are not used in the entire code.
time, sys, os, traceback, optparse, also rom copy import copy, deepcopy
these are never used. So should they be cleaned up?